### PR TITLE
docs: change Compositor to plain text

### PR DESCRIPTION
**Pull request by JounQin**
This doc issue is introduced at #1387

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Support this effort and give back by sponsoring on [OpenCollective][collective]!
 
 ## Authors
 
-- [John Otander][john] ([@4lpine][4lpine]) – [Components AI](https://components.ai)
+- [John Otander][john] ([@4lpine][4lpine]) – [Components AI][]
```
Comment (ChristianMurphy) [inline readme.md:143]: :+1:

File: readme.md [hunk 2]
```diff
@@ -164,7 +164,7 @@ Join us on [GH Discussions][chat]!
 
 ## License
 
-[MIT][] © [Compositor][] and [Vercel][]
+[MIT][] © [Components AI][] and [Vercel][]
```
Comment (ChristianMurphy) [inline readme.md:167]: did the license transfer? :thinking: 
/cc @johno @wooorm

File: readme.md [hunk 3]
```diff
@@ -184,6 +184,7 @@ Join us on [GH Discussions][chat]!
 [rauchg]: https://twitter.com/rauchg
 [timneutkens]: https://twitter.com/timneutkens
 [jxnblk]: https://twitter.com/jxnblk
+[components ai]: https://components.ai
```
Comment (ChristianMurphy) [inline readme.md:187]: :+1:

**Review by ChristianMurphy — COMMENTED** [review]
Thanks @JounQin!

File: readme.md (hunk 2 above)
Comment (wooorm) [reply readme.md:167]: Copyright doesn’t transfer like this. Compositor !== Components AI as far as I can tell, so this change is incorrect.
We can drop the link to `Compositor` though, if it’s no longer alive

Comment (JounQin) [reply readme.md:167]: OK, I'll remove it then.

Comment (JounQin) [reply readme.md]: Done!

File: readme.md [hunk 4]
```diff
@@ -164,7 +164,7 @@ Join us on [GH Discussions][chat]!
 
 ## License
 
-[MIT][] © [Compositor][] and [Vercel][]
+[MIT][] © [Vercel][]
```
Comment (wooorm) [inline readme.md:167]:
```suggestion
[MIT][] © Compositor and [Vercel][]
```

I meant remove the link, not remove the text. Sorry.
Copyright can’t just be removed either!

**Review by ChristianMurphy — APPROVED** [review]

**Pull request merged**